### PR TITLE
feat(account): add account deletion request

### PR DESCRIPTION
**Pull request by kamal09rumba**
- Add user account deletion support
- Update README

### CODE REVIEW

File: support/types.py [hunk 1]
```diff
@@ -10,6 +11,13 @@
 )
 
 
+class AccountDeletionRequestType(DjangoObjectType):
+    class Meta:
+        model = AccountDeletionRequest
+        description = "Type defincation for account deletion request"
```
Comment (timilsinabishal) [inline support/types.py]: defination

File: support/mutations.py [hunk 2]
```diff
@@ -1,8 +1,40 @@
...
+from support.serializers import AccountDeletionRequestSerializer, FeedbackSerializer
+from user.models import User
 
 
 class FeedbackMutation(SerializerMutation):
     class Meta:
         serializer_class = FeedbackSerializer
+
+
+class AccountDeletionRequestMutation(SerializerMutation):
```
Comment (timilsinabishal) [inline support/mutations.py:13]: is error format fixed?

File: support/types.py (hunk 1 above)
Comment (kamal09rumba) [reply support/types.py]: changed

File: support/mutations.py (hunk 2 above)
Comment (kamal09rumba) [reply support/mutations.py:13]: resolved

**Pull request merged**